apps/site/assets/ts/hooks/usePredictionsChannel.ts

```diff
@@ -88,28 +81,21 @@ export const groupByHeadsigns = (
 const usePredictionsChannel = (
   routeId: string,
   stopId: string,
-  directionId: 0 | 1,
-  numPredictions?: number
+  directionId: 0 | 1
 ): PredictionsByHeadsign => {
   const channelName = `predictions:${routeId}:${stopId}:${directionId}`;
-  const reducer: Reducer<
-    PredictionsByHeadsign,
-    ChannelPredictionResponse
-  > = useCallback(
-    (oldGroupedPredictions, { predictions }) => {
-      const parsedPredictions = predictions.map(parsePrediction);
-      const newGroupedPredictions = groupByHeadsigns(
-        parsedPredictions,
-        numPredictions
-      );
-      // don't attempt to reconcile with prior predictions, just replace state with
-      // all the new predictions from the channel if there are any changes.
-      return deepEqual(oldGroupedPredictions, newGroupedPredictions)
-        ? oldGroupedPredictions
-        : newGroupedPredictions;
-    },
-    [numPredictions]
-  );
+  const reducer: Reducer<PredictionsByHeadsign, ChannelPredictionResponse> = (
```

Comment:
The `useCallback` was removed because it memoized the `numPredictions` as an input.  Since that param is no longer in use the use of `useCallback` became obsolete.